llvm/docs/ReleaseNotes.rst

```diff
@@ -359,6 +371,8 @@ Changes to LLDB
   fields are present, however this is not always possible or entirely
   accurate. If in doubt, refer to the numerical value.
 
+* On Windows, LLDB can now read the thread names.
```

Comment:
@oltolm Is this accurate for 95e5839e06fdffd278499257c6e7679bba3d6868?

Reply:
Yes, in the sense that if a thread has a name, it will display it.